frontend/src/components/layout/MainNavigation.tsx

```diff
@@ -17,9 +17,9 @@ function MainNavigation() {
             />{" "}
             Tiden
           </Navbar.Brand>
-          {/*<Nav>*/}
-          {/*  <Nav.Link href="/dashboard">Dashboard</Nav.Link>*/}
-          {/*</Nav>*/}
+          <Nav.Link href="/dashboard" style={{ color: "grey" }}>
```

Comment:
let's keep this one out so that it doesn't show up on the landing page

Reply:
omitted 👍🏻